tests/containers/podman_pods.pm

```diff
@@ -81,11 +81,17 @@ sub run {
         install_k3s();
         record_info('Test', 'kube apply');
         assert_script_run('podman kube apply --kubeconfig ~/.kube/config -f pod.yaml');
-        assert_script_run('kubectl wait --for=condition=Ready pod/testing-pod');
+        assert_script_run('kubectl wait --for=condition=Ready pod/testing-pod', timeout => 240);
```

Comment:
```suggestion
        assert_script_run('kubectl wait --for=condition=Ready pod/testing-pod --timeout=240s');
```

nit: If doing so would not hamper the test flow

Reply:
The default for `assert_script_run()` is `90`. Should I use both `--timeout=240s` as well as `, timeout => 245`?

Reply:
Quite the oposit. The `kubectl` must end before `assert_script_run()`. Otherwise we won't know if the `kubectl` was just slow or if it had an issue.

Reply:
Right, but what I meant was, if the default value for kubectl is something like 60s and you expect it to complete within ~120s (assuming this because you've set the wrappers to 240s), it would actually be detrimental to your workflow because nowhere would this 60s be present in this snippet. 

But looks like the default behavior is to check once and not wait[1] which I guess has been working fine for you in the past so you _could_ either overlook this suggestion or add it for posterity just to make the expectations clear.

[1] - https://github.com/kubernetes/kubectl/blob/b3d63dd09bcd124f198954e2cf8a43b3c68de207/pkg/cmd/wait/wait.go#L147

Reply:
I hope that right now it's clear in the code. The `kubectl wait` waits for `240s` and `assert_script_run` for `260` so there is enough time for the `kubectl wait` to properly exit.

Thank you for your hint!

Reply:
That sounds perfect!